src/lkl/setup.c

```diff
@@ -469,15 +459,12 @@ static void* lkl_create_crypto_disk_thread(struct lkl_crypt_device* lkl_cd)
 
     // As we generate our own key and don't use a simple "password" we use
     // the minimal kdf settings possible.
-    struct crypt_pbkdf_type pbkdf =
-    {
+    struct crypt_pbkdf_type pbkdf = {
         .type = "pbkdf2",
         .hash = "sha256",
         .iterations = MIN_LUKS2_ITERATIONS,
     };
-    struct crypt_params_luks2 params =
-    {
-        .sector_size = SECTOR_SIZE,
+    struct crypt_params_luks2 params = {.sector_size = SECTOR_SIZE,
```

Comment:
I wasn't going to complain this time about a PR that mixes formatting with functional changes, but in this case the formatting changes are making the formatting less correct.  Please submit this PR with only the functional changes.

Reply:
I just reverted all those changes before committing. Weird.